manual_tests/internal/create_data_action.go

```diff
@@ -0,0 +1,59 @@
+package internal
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/bsv-blockchain/go-sdk/transaction"
+	sdk "github.com/bsv-blockchain/go-sdk/wallet"
+	"github.com/bsv-blockchain/go-wallet-toolbox-manual-tests/internal/fixtures"
+	"github.com/go-softwarelab/common/pkg/to"
+)
+
+func (m *Manager) CreateActionWithData(user fixtures.UserConfig, data string) (string, fixtures.Summary, error) {
+	var summary fixtures.Summary
+
+	summary = append(summary, fmt.Sprintf("Using wallet for user %s", user.Name))
```

Comment:
```suggestion
	summary := fixtures.Summary{fmt.Sprintf("Using wallet for user %s", user.Name)}
```
or even you could do 
```go
summary := fixtures.Summary{
	fmt.Sprintf("Using wallet for user %s", user.Name),
	fmt.Sprintf("Creating data output with data: %s", data),
}
```

i think is shorter and more readable that u defining slice ...